src/main/java/com/github/manolo8/darkbot/modules/utils/PortalJumper.java

```diff
@@ -44,15 +47,22 @@ public void jump(Portal target) {
             if (!gm.group.isValid() || gm.group.size < minGroupSize) return;
         }
 
-        hero.jumpPortal(target);
-
         if (target != last) {
             last = target;
+            tryingToJumpSince = System.currentTimeMillis();
             nextMoveClick = System.currentTimeMillis() + 5000;
         } else if (System.currentTimeMillis() > nextMoveClick && !target.clickable.enabled) {
             hero.drive.clickCenter(true, target.locationInfo.now);
             nextMoveClick = System.currentTimeMillis() + 10000;
         }
+
+        if (tryingToJumpSince != 0 && System.currentTimeMillis() > tryingToJumpSince + 120000) {
```

Comment:
You probably want this as an else-if from the above, so if the desync is known (clickable isn't enabled, so you need to still travel towards it) it won't be trying to refresh

Reply:
clickable is always enabled on desync since it happens after jump(probably) check both screenshots in bug report

Reply:
I'm aware of the specific bug you're trying to solve, my point is precisely that because clickable IS enabled, you should only do this on after an `else` from above instead of always

Reply:
merged to else if chain